bindings/azure/blobstorage/blobstorage.go

```diff
@@ -135,9 +136,25 @@ func (a *AzureBlobStorage) Invoke(req *bindings.InvokeRequest) (*bindings.Invoke
 	}
 
 	// Unescape data which will still be a JSON string
-	unescapedData, _ := strconv.Unquote(string(req.Data))
+	unescapedData, unescapeError := strconv.Unquote(string(req.Data))
 
-	_, err := azblob.UploadBufferToBlockBlob(context.Background(), []byte(unescapedData), blobURL, azblob.UploadToBlockBlobOptions{
+	if unescapeError != nil {
+		return nil, unescapeError
+	}
+
+	var data []byte
```

Comment:
```suggestion
	data := []byte(unescapedData)
```
(avoiding an uninitialized var)